public/app/views/classifications/_related_listing.html.erb

```diff
@@ -1,13 +1,12 @@
 <ul class="classifications_list">
   <% classifications.each do |c| %>
     <li>
-      <ul class="breadcrumb">
-      <% c.fetch('breadcrumb').each do |crumb, i| %>
+      <% c.fetch('breadcrumb').each_with_index do |crumb, index| %>
         <% crumb_uri = crumb.fetch(:uri)
            crumb_uri = c.fetch('uri') if crumb_uri.blank? %>
-        <li><%= link_to crumb.fetch(:crumb), app_prefix(crumb_uri) %></li>
+        <%= link_to crumb.fetch(:crumb), app_prefix(crumb_uri) %>
+        <%= '/' unless index == c.fetch('breadcrumb').length - 1 %>
```

Comment:
@Vchouliaras instead of hard coding the `'/'` here on line 8, you'll want to use the i18n for `context_delimiter` (see [an example](https://github.com/archivesspace/archivesspace/blob/master/public/app/views/shared/_result_breadcrumbs_ancestor.html.erb#L2)).

Reply:
Updated